content/en/ecosystem/adopters.md

```diff
@@ -6,28 +6,36 @@ spelling: cSpell:ignore Dapr Datenrettungsdienste Globale Logicmonitor Logz Wand
 # All spelling entries must be on a single line
 ---
 
-A non-exhaustive, alphabetized list of
+OpenTelemetry's mission is to to enable effective observability for all its
+end-users. If you thinking about adopting OpenTelemetry for your organization,
+you may be curious how other adoption journeys looked like. On this page you can
+find a non-exhaustive list of
 [end-user organizations](https://community.cncf.io/end-user-community/) that
-have adopted OpenTelemetry. Also see [Vendors](../vendors/).
-
-<!-- prettier-ignore-start -->
-Organization                                      | Components                      |  Learn more
-------------------------------------------------  | ------------------------------- |  ----------
-[AppDirect](https://www.appdirect.com/)           | Collector                       |
-[Care.com](https://www.care.com)                  | Collector, Go, Java, JS, .NET   |
-[Cloud Scale](https://www.cloudscaleinc.com)      | Collector, Python    |
-[EcoBee](https://www.ecobee.com/)                 | Collector, Java                 |  [blog post](https://www.honeycomb.io/blog/bees-working-together-how-ecobees-engineers-adopted-honeycomb-for-visibility-into-system-optimization-and-customer-experience)
-[Farfetch](https://www.farfetch.com/)             | Collector, Java, .NET, Node.js, Python, Operator | [blog post](/blog/2023/end-user-q-and-a-03/)
-[F5](https://www.f5.com/)                         | Collector.                      |
-[OrderMyGear](https://www.ordermygear.com/)       |                                 |
-[PITS Globale Datenrettungsdienste](https://www.pitsdatenrettung.de/) | Python      |
-[Shopify](https://www.shopify.com/)               | Collector, Go, Ruby             |
-[Transit](https://transitapp.com/)                |                                 |
-[Uplight](https://uplight.com/)                   | Collector, Ruby, Java, Python, .NET | [blog post](/blog/2023/end-user-q-and-a-02/)
-[Wandera](https://www.wandera.com/)               | Collector                       |
-[Zocdoc](https://www.zocdoc.com/)                 |                                 |
-<!-- prettier-ignore-end -->
-
-To add your organization, edit this page and submit a PR. You will need to
-include a link to a blog post that describes how your organization makes use of
-OTel.
+have adopted OpenTelemetry for
+[Observability](/docs/concepts/observability-primer/).
+
+{{% ecosystem/adopters-table %}}
+
+To have your organization included,
+[add an entry to the list](https://github.com/open-telemetry/opentelemetry.io/tree/main/data/ecosystem/adopters.yaml)
+and submit a PR.
+
+You will need to include[^existing-entries]:
+
+- a link to a blog post that describes how your organization makes use of
+  OpenTelemetry.
+- a point of contact (GitHub handle or email address), we can reach out for
+  future inquiries.
```

Comment:
```suggestion
- Link to a blog post that describes how your organization makes use of
  OpenTelemetry
- GitHub handle or email address as a point of contact so that we can reach out in case we have questions
```